superset/dashboards/commands/update.py

```diff
@@ -51,6 +52,12 @@ def run(self) -> Model:
         try:
             dashboard = DashboardDAO.update(self._model, self._properties, commit=False)
             dashboard = DashboardDAO.update_charts_owners(dashboard, commit=True)
+            if self._properties.get("json_metadata") is not None:
+                dashboard = DashboardDAO.set_dash_metadata(
+                    dashboard,
+                    data=json.loads(self._properties.get("json_metadata", "{}")),
```

Comment:
If for some reason `json_metadata` is an empty string this would raise an error:

```python
>>> properties = {'json_metadata': ''}
>>> properties.get('json_metadata') is not None
True
>>> json.loads(properties.get("json_metadata", "{}"))
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

You might wanna do instead:

```suggestion
            if self._properties.get("json_metadata"):
                dashboard = DashboardDAO.set_dash_metadata(
                    dashboard,
                    data=json.loads(self._properties.get("json_metadata", "{}")),
```